patches/04-plugin-openshift-v1

```diff
@@ -0,0 +1,147 @@
+@@ -81,7 +81,7 @@ spec:
+                 - --upstream=http://127.0.0.1:8080/
+                 - --logtostderr=true
+                 - --v=10
+-                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
++                image: quay.io/openshift/origin-kube-rbac-proxy:4.7.0
+                 name: kube-rbac-proxy
+                 ports:
+                 - containerPort: 8443
+diff -up ./internal/generate/testdata/clusterserviceversions/output/with-ui-metadata.clusterserviceversion.yaml.plugin-openshift-v1 ./internal/generate/testdata/clusterserviceversions/output/with-ui-metadata.clusterserviceversion.yaml
+--- ./internal/generate/testdata/clusterserviceversions/output/with-ui-metadata.clusterserviceversion.yaml.plugin-openshift-v1	2021-01-14 17:38:53.418288715 -0800
++++ ./internal/generate/testdata/clusterserviceversions/output/with-ui-metadata.clusterserviceversion.yaml	2021-01-14 17:39:01.016404930 -0800
+@@ -125,7 +125,7 @@ spec:
+                 - --upstream=http://127.0.0.1:8080/
+                 - --logtostderr=true
+                 - --v=10
+-                image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
++                image: quay.io/openshift/origin-kube-rbac-proxy:4.7.0
+                 name: kube-rbac-proxy
+                 ports:
+                 - containerPort: 8443
+diff -up ./internal/generate/testdata/go/static/basic.operator.yaml.plugin-openshift-v1 ./internal/generate/testdata/go/static/basic.operator.yaml
+--- ./internal/generate/testdata/go/static/basic.operator.yaml.plugin-openshift-v1	2021-01-14 17:38:53.419288730 -0800
++++ ./internal/generate/testdata/go/static/basic.operator.yaml	2021-01-14 17:39:01.016404930 -0800
+@@ -266,7 +266,7 @@ spec:
+         - --upstream=http://127.0.0.1:8080/
+         - --logtostderr=true
+         - --v=10
+-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
++        image: quay.io/openshift/origin-kube-rbac-proxy:4.7.0
+         name: kube-rbac-proxy
+         ports:
+         - containerPort: 8443
+diff -up ./test/ansible/config/default/manager_auth_proxy_patch.yaml.plugin-openshift-v1 ./test/ansible/config/default/manager_auth_proxy_patch.yaml
+--- ./test/ansible/config/default/manager_auth_proxy_patch.yaml.plugin-openshift-v1	2021-01-14 17:38:53.420288745 -0800
++++ ./test/ansible/config/default/manager_auth_proxy_patch.yaml	2021-01-14 17:39:01.016404930 -0800
+@@ -10,7 +10,7 @@ spec:
+     spec:
+       containers:
+       - name: kube-rbac-proxy
+-        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.5.0
++        image: quay.io/openshift/origin-kube-rbac-proxy:4.7.0
+         args:
+         - "--secure-listen-address=0.0.0.0:8443"
+         - "--upstream=http://127.0.0.1:8080/"
+diff -up ./testdata/ansible/memcached-operator/bundle/manifests/memcached-operator.clusterserviceversion.yaml.plugin-openshift-v1 ./testdata/ansible/memcached-operator/bundle/manifests/memcached-operator.clusterserviceversion.yaml
```

Comment:
Contents of this file from this point to its end are only necessary if testdata is not regenerated on merge. @jmrodri you said this is the case correct?

Reply:
Yes, we do NOT regenerate the testdata downstream.